### DISCUSSION TITLE
Sub-committee self-selection and self-organization

### DISCUSSION THREAD

**Discussion by hidyverse** [resbazaz/festival2024#7] [General]
Onwards!

Based on our kickoff, I have voluntold a few organizers as leaders of sub-committees. Please find below those folks tagged here along with some major tasks of each sub-committee.

Let's go ahead and self-assign to these sub-committees as we are interested/available and self-organize as sub-committees to move forward with our planning.

:clapper: **Speakers and Helpers**: [@Ethan Jahn](https://resbazaz.slack.com/team/U06FU9UU211)

Work with scheduling to recruit speakers
Recruit helpers

:ticket: **Scheduling**: YOU !
Work with Logisitics and Social to determine allowable schedule
Work with Speakers and Helpers to fill schedule as events are available

 :loud_sound: **Advertisment**: [@Laura W Dozal](https://resbazaz.slack.com/team/U01MLT6R2UD)

Work with Logistics and Social, Scheduling, and Speakers and Helpers for conference info
Post ResBaz Festival 24 UA calendars, ssocial media, [Meetup.com](http://meetup.com/), and other locations
Email listservs and past conference attendees
Work with Website to post festival
 Create conference logos, fliers, and other promotional materials

👯‍♂️ **Logistics and Social:** [@Brandon](https://resbazaz.slack.com/team/U03PE0R0S3G)

Book venue for conference and social
Decide on virtual venue
Create registration page

:spider_web: **Website**: [@Alex Bigelow](https://resbazaz.slack.com/team/UPUP78S75)

Create event webpage

**Comment by julianpistorius** [comment]
I'll volunteer to work with @alex-r-bigelow on the website.

I can also help with tweets, and delegate access to the Twitter account.

**Comment by kepplerm** [comment]
I'm happy to help with logistics and the registration page (or anything else for which help is needed).

**Reply by brandon-jernigan** [reply]
Awesome, thank you!

**Comment by julianpistorius** [comment]
Thanks to @hidyverse & @val-pf for organizing and leading this morning's meeting. 

Also thank you to the other committee members and organizers: @alex-r-bigelow @lwdozal @brandon-jernigan @kepplerm @e-eight 

See [notes here](https://hackmd.io/@julianp-codecove/rkVA0lD96#Follow-up-Kickoff-Meeting-2024-Mar-05).

Could y'all please create GitHub issues for yourselves for any action items assigned to you?

**Comment by brandon-jernigan** [comment]
[like]  Jernigan, Brandon - (brandonje... reacted to your message:
________________________________
From: Julian Pistorius ***@***.***>
Sent: Tuesday, March 5, 2024 6:50:47 PM
To: resbazaz/festival2024 ***@***.***>
Cc: Jernigan, Brandon - (brandonjernigan) ***@***.***>; Mention ***@***.***>
Subject: [EXT]Re: [resbazaz/festival2024] Sub-committee self-selection and self-organization (Discussion #7)


External Email

Thanks to @hidyverse<https://github.com/hidyverse> & @val-pf<https://github.com/val-pf> for organizing and leading this morning's meeting.

Also thank you to the other committee members and organizers: @alex-r-bigelow<https://github.com/alex-r-bigelow> @lwdozal<https://github.com/lwdozal> @brandon-jernigan<https://github.com/brandon-jernigan> @kepplerm<https://github.com/kepplerm> @e-eight<https://github.com/e-eight>

See notes ***@***.***/rkVA0lD96#Follow-up-Kickoff-Meeting-2024-Mar-05>.

Could y'all please create GitHub issues for yourselves for any action items assigned to you?
